content/en/docs/what-is-opentelemetry.md

```diff
@@ -39,7 +39,7 @@ OpenTelemetry consists of the following major components:
 - Various other tools, such as the
   [OpenTelemetry Operator for Kubernetes](/docs/kubernetes/operator/)
 
-OpenTelemetry is compatible with a wide variety of open source
+OpenTelemetry is compatible with a wide variety of
 [ecosystem integrations](/ecosystem/integrations/).
 
 OpenTelemetry is supported by a vast number of [vendors](/ecosystem/vendors/),
```

Comment:
Rather than "vast" which is a weasel word, how about we say 60+ or the like?

Reply:
```suggestion
OpenTelemetry is supported by 40+ [vendors](/ecosystem/vendors/),
```

like that? I counted the ones on the page and got to ~47